src/renderer/screens/dashboard/index.js

```diff
@@ -67,21 +66,6 @@ const DashboardPage = ({ saveSettings }: Props) => {
     [saveSettings],
   );
 
-  const Header = useCallback(
-    ({ portfolio }) => (
-      <BalanceInfos
-        t={t}
-        unit={counterValue.units[0]}
-        isAvailable={portfolio.balanceAvailable}
-        since={selectedTimeRange}
-        valueChange={portfolio.countervalueChange}
-        totalBalance={portfolio.balanceHistory[portfolio.balanceHistory.length - 1].value}
-        handleChangeSelectedTime={handleChangeSelectedTime}
-      />
-    ),
-    [t, counterValue.units, selectedTimeRange, handleChangeSelectedTime],
-  );
```

Comment:
well yeah that definitely sounded like a hacky idea

Reply:
Yep that was a bit mind blowing